akka-stream/src/main/boilerplate/akka/stream/javadsl/BidiFlowCreate.scala.template

```diff
@@ -13,14 +13,14 @@ trait BidiFlowCreate {
   import language.implicitConversions
   private implicit def p[A, B](pair: Pair[A, B]): (A, B) = pair.first -> pair.second
 
-  def create[I1, O1, I2, O2](block: japi.Function[FlowGraph.Builder, BidiShape[I1, O1, I2, O2]]): BidiFlow[I1, O1, I2, O2, Unit] =
+  def create[I1, O1, I2, O2](block: japi.Function[FlowGraph.Builder[Unit], BidiShape[I1, O1, I2, O2]]): BidiFlow[I1, O1, I2, O2, Unit] =
```

Comment:
This is an input that we give to the user, so we need to give the proper type (which is Unit). We should be flexible in places where we accept input from the user, where `_` may be an option for Java (needs verification). We must not ever return such a wildcard type to users.